backend/open_webui/routers/ollama.py

```diff
@@ -866,6 +869,14 @@ async def embed(
 
         model = form_data.model
 
+        configs = request.app.state.config.OLLAMA_API_CONFIGS
+        prefix_ids = [config.get('prefix_id') for config in configs.values() if config.get('prefix_id')]
+        prefix_id = None
+        for prefix in prefix_ids:
+            if model.startswith(prefix):
```

Comment:
I thought about it afterward, but adding a period here would make it more resilient.
This way, potential conflicts of prefixes that are substrings of each other are avoided:

`if model.startswith(f"{prefix}."):`